contrib/natss/pkg/dispatcher/dispatcher/dispatcher.go

```diff
@@ -67,8 +68,20 @@ func createReceiverFunction(s *SubscriptionsSupervisor, logger *zap.SugaredLogge
 		logger.Infof("Received message from %q channel", channel.String())
 		// publish to Natss
 		ch := getSubject(channel)
+		if s.natssConn == nil {
```

Comment:
Save `s.natssConn` to a local variable so that all the checks are done against the same value (i.e. we check for `nil` and then use the checked object).